feature-detects/elem/ruby.js

```diff
@@ -8,7 +8,7 @@
   "authors": ["Cătălin Mariș"],
   "notes": [{
     "name": "WHATWG Specification",
```

Comment:
total nitpicking: since you unified the w3c naming, maybe you can do it with WHATWG / WhatWG Spec /Specification too?

Reply:
I agree with this comment. I will try to do this for other links as well.